### PR TITLE
[DO NOT MERGE] etools-ajax v2 demo

### PR DESCRIPTION
**Pull request by daniel-tabarcea**
- retrieval and saving works
- new partner not working at the moment

### CODE REVIEW

File: pmp/app-elements/partners/partners-data/partner-item-data.html [hunk 1]
```diff
@@ -32,18 +32,20 @@
...
           },
-          withUpload: Boolean,
-          prepareMultipartData: Boolean,
+
+          options: {
+            type: Object,
+            value: {
+              endpoint: null
+            },
+            csrf: true
```
Comment (adi130987) [inline pmp/app-elements/partners/partners-data/partner-item-data.html]: `csrf` must be inside value object and should be used only for unsafe requests

File: pmp/app-elements/etools-ajax-v2/etools-ajax.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
+<link rel="import" href="../../../bower_components/polymer/polymer.html">
+<link rel="import" href="../../../bower_components/etools-dexiejs/dexie.html">
```
Comment (adi130987) [inline pmp/app-elements/etools-ajax-v2/etools-ajax.html]: no need for dexie import here, the import it's already made in app config, but I think we should include it in EtoolsAjaxCacheBehavior

File: pmp/app-elements/etools-ajax-v2/etools-ajax.html [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+          }.bind(this))
+          .catch(function (error) {
+            this.handleError(error);
+          }.bind(this));
+        },
+
+        _optionsChanged: function(options) {
+          this.debounce('etools-ajax-options-changed', function() {
+            this.send();
+          }.bind(this), 300)
```
Comment (adi130987) [inline pmp/app-elements/etools-ajax-v2/etools-ajax.html]: add debounce time in properties with a default value

File: pmp/app-elements/etools-ajax-v2/etools-ajax.html [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+        },
+
+        handleError: function (error) {
+          if (error.status === 401) {
+            this.fire('unauthorized', {error: error});
+            return;
```
Comment (robertavram) [inline pmp/app-elements/etools-ajax-v2/etools-ajax.html]: remove returns

File: pmp/app-elements/etools-ajax-v2/etools-ajax.html [hunk 5]
```diff
@@ -0,0 +1,63 @@
+<link rel="import" href="../../../bower_components/polymer/polymer.html">
```
Comment (robertavram) [inline pmp/app-elements/etools-ajax-v2/etools-ajax.html]: abstract element as bower component

**Pull request closed without merging**